ci/continuous-builder.sh

```diff
@@ -3,45 +3,79 @@
 # AliRoot / AliPhysics. Notice this will do an incremental build, not a full
 # build, so it really to catch errors earlier.
 
-# timeout vs. gtimeout (macOS with Homebrew)
-TIMEOUT_EXEC=timeout
-type $TIMEOUT_EXEC > /dev/null 2>&1 || TIMEOUT_EXEC=gtimeout
-function short_timeout () { $TIMEOUT_EXEC -s9 "$TIMEOUT" "$@"; }
-function long_timeout () { $TIMEOUT_EXEC -s9 "$LONG_TIMEOUT" "$@"; }
-
 . build-helpers.sh
 
-if ! [ -d ali-bot ]; then
-  git clone https://github.com/alisw/ali-bot
-fi
+if [ "$1" != --skip-setup ]; then
+  for var in GITHUB_TOKEN GITLAB_USER GITLAB_PASS AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS_KEY INFLUXDB_WRITE_URL ALIBOT_ANALYTICS_ID MONALISA_HOST MONALISA_PORT MESOS_ROLE CONTAINER_IMAGE; do
+    if [ -z "${!var}" ]; then
+      echo "$(basename "$0"): error: required variable $V not defined!" >&2
+      exit 1
+    else
+      export "${var?}"
+    fi
+  done
+
+  case "$(uname)" in
+    # The MacOS builders aren't in the right directory at first, apparently.
+    Darwin) ci_work_dir=/build/ci_checks/alice_ci_$WORKER_INDEX
+            mkdir -p "$ci_work_dir"
+            cd "$ci_work_dir" || exit 1;;
+  esac
```

Comment:
This should be solved by how we start it on mac, I would rather avoid extra stuff in here for something that is really usage specific.